MathSoc/mathsoc-website
# Cartoons Archive: Frontend

Develops the cartoons archive.

![image](https://github.com/MathSoc/mathsoc-website/assets/37753525/c9737105-8aa3-4f21-b701-3ca265e95ec5)

How to test:
* Checkout the branch locally.
* Try running the server. It shouldn't crash.
* Upload some images to the img/cartoons/uploads folder in the directory structure specified in CartoonsController (`department/coursecode/Title with spaces and capitalization/<several imgs>`).  Then reload the server to allow the JSON file to reset.  You should see the cartoons archive page populate.
